website/views/options/social/index.jade

```diff
@@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
   .tab-content
     .tab-pane.active
       div(ui-view)
+
+script(type='text/ng-template', id='modals/copyChatToDo.html')
```

Comment:
Not something that needs to be done before this gets merged in, but it would be nice to have a preview so you can see what your todo will look like. Maybe with a popover always showing?